.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+- [ ] **Submitter**: Include the JIRA issue number in the PR description
+- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
+- [ ] **Submitter**: If you're adding/switching libraries or otherwise changing the design, update the [design documentation](https://broadinstitute.atlassian.net/wiki/pages/viewpage.action?pageId=114531509).
+- [ ] **Submitter**: JIRA ticket checks:
+  * Acceptance criteria exists and is met
+  * Note any changes to implementation from the description
+  * Add notes on what you've tested
+* Review cycle:
+  * Team may comment on PR at will
+  * **Reviewer assigns to submitter** for feedback fixes
+  * Submitter rebases to develop again if necessary
+  * Submitter makes further commits. DO NOT SQUASH
+  * Submitter updates documentation as needed
+  * Submitter **reassigns to reviewer** for further feedback
+- [ ] **Submitter**: Verify all tests go green, including CI tests
+- [ ] **Submitter**: Squash commits and merge to develop
+- [ ] **Submitter**: Delete branch after merge
+- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
```

Comment:
Would there be another step here where after you test it on Dev you move it to staging and move ticket to QA?